src/modules/StorageProposal.rb

```diff
@@ -668,7 +668,7 @@ def try_add_boot(conf, disk, force)
         pb = {}
 	pb["mount"] = Partitions.BootMount
 	pb["size"] = Partitions.ProposedBootsize
-	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot
+	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot && !is_dasd?(disk.fetch("device", ""))
```

Comment:
I looked at the code again and the ```size``` there is in B, not KiB. So it's at least 200 KiB, not 200 MiB.

That also means the ```- 1024``` should not be ```/ 1024```. Maybe it's still a typo that just doesn't hurt. Maybe it should be removed. Who knows.